boards/arm/nucleo_l552ze_q/doc/nucleol552ze_q.rst

```diff
@@ -169,12 +169,12 @@ input/output, pull-up, etc.
 
 Available pins:
 ---------------
-.. image:: img/nucleo_l552ze_q_arduino.png
+.. image:: img/nucleo_l552ze_q_zio_left_2020_2_11.png
    :width: 720px
    :align: center
    :height: 540px
    :alt: Nucleo L552ZE Q Arduino connectors
```

Comment:
Comment doesn't correspond any more to the image. It is not Arduino connector vs Morpho connector, but rather left vs right side

Reply:
Good catch! Fixed, txs